### PR TITLE
add pya init command

### LINKED ISSUE #1390: add `pya init` command to cli to init `~/MyPyaerocom` path
**Is your feature request related to a problem? Please describe.**
for external usage users need to adjust the paths settings coming as default from 
`https://github.com/metno/pyaerocom/blob/main-dev/pyaerocom/data/paths.ini`.
The path `~/MyPyaerocom` is only created on the first import of pyaerocom. To make things easy for the users, it would be nice to have an easy way of speeding that up.

**Describe the solution you would like to see**
Intriduce a command `pya init` that creates the path `~/MyPyaerocom` and maybe also copies the default `paths.ini` there

### PR DESCRIPTION
**Pull request by jgriesfeller**
## Change Summary

The title says it all
## Related issue number
fix #1401 

## Checklist

* [x] Start with a draft-PR
* [x] The PR title is a good summary of the changes
* [x] PR is set to AeroTools and a tentative milestone
* [x] Documentation reflects the changes where applicable
* [x] Tests for the changes exist where applicable
* [x] Tests pass locally
* [x] Tests pass on CI
* [x] At least 1 reviewer is selected
* [x] Make PR ready to review

### CODE REVIEW

File: pyaerocom/scripts/cli.py [hunk 1]
```diff
@@ -106,5 +107,29 @@ def getsampledata(
     download_minimal_dataset(extract_dir_override=extract_dir)
 
 
+@main.command()
+def init():
+    """init ~/MyPyaerocom directory and copy the default paths.ini there"""
+
+    mypyaerocom = Path.home() / "MyPyaerocom"
+    if not Path.exists(mypyaerocom):
+        Path.mkdir(mypyaerocom)
```
Comment (thorbjoernl) [inline pyaerocom/scripts/cli.py]:
```suggestion
    mypyaerocom.mkdir(exist_ok=True)
```

File: pyaerocom/scripts/cli.py [hunk 2]
```diff
@@ -106,5 +107,29 @@ def getsampledata(
...
+
+    # copy default paths.ini if it doesn't exit
+    ini_in_path = Path(__file__).parents[1].joinpath("data/paths.ini")
+    ini_out_path = Path.joinpath(mypyaerocom,"paths.ini")
```
Comment (thorbjoernl) [inline pyaerocom/scripts/cli.py]:
```suggestion
    ini_out_path = mypyaerocom / "paths.ini"
```

**Review by thorbjoernl — APPROVED** [review]
Minor suggestions, otherwise LGTM

File: pyaerocom/scripts/cli.py (hunk 1 above)
Comment (jgriesfeller) [reply pyaerocom/scripts/cli.py]: changed

File: pyaerocom/scripts/cli.py (hunk 2 above)
Comment (jgriesfeller) [reply pyaerocom/scripts/cli.py]: changed

**Pull request merged**